### PR TITLE
fix_motor_css

### PR DESCRIPTION
**Pull request by haslinghuis**
Thanks to @hydra for pointing out the bug with the width of some elements.

- [x] Fixes width of Mixer selector
- [x] Fixes width and margin of ESC protocol selector
- [x] Changed selectors by id which didn't work on Windows.
- [x] Thanks to @limonpsb also fixed horizontal scrollbar.

image

### CODE REVIEW

**Review by McGiverGim — DISMISSED** [review]

**Comment by hydra** [comment]
I checked out your branch and confirm it's fixed.

image

**Comment by haslinghuis** [comment]
@limonspb didn't see my comment on Slack - so we now have another fix here: https://github.com/betaflight/betaflight-configurator/pull/2854

**Comment by Asizon** [comment]
This fix breaks other things around motor tab @haslinghuis, prefer to go with limons PR fix :)

**Review by limonspb — CHANGES_REQUESTED** [review]
@haslinghuis 
just removing min-width is not a full fix of the problem that I introduced earlier :)
Can check the code in my PR.
Here are the problems here:
image

**Comment by limonspb** [comment]
> This fix breaks other things around motor tab @haslinghuis, prefer to go with limons PR fix :)

lets keep this PR because it was created earlier than mine. Lets just add other fixes to it.
Im closing my PR.

**Comment by limonspb** [comment]
whatever works :)
Can update this PR or use mine :) Let me know!

**Comment by Asizon** [comment]
@limonspb dont worry about it, if you have modified this tab and code in earlier PR you know what happens here.

**Comment by limonspb** [comment]
> @limonspb dont worry about it, if you have modified this tab and code in earlier PR you know what happens here.

It is a little funny that in my PR that breaks this selects you can see the problem even on the screenshot i provided LOL. My bad guys.
https://github.com/betaflight/betaflight-configurator/pull/2848

**Comment by haslinghuis** [comment]
@limonspb fixed

**Comment by limonspb** [comment]
> @limonspb @Asizon fixed (see updated picture at the beginning of the PR)

i don't think that 
`overflow-x: hidden;`
is the right choice. It's better to figure out what causes overflow and fix that.

**Comment by haslinghuis** [comment]
Done 😃

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/css/tabs/motors.css
Comment (Asizon) [inline src/css/tabs/motors.css:67]: I see that the esc protocols have some unnecessary extra whitespace, it is unsightly for a selector that adapts to the length of the names:

image

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/css/tabs/motors.css
Comment (Asizon) [reply src/css/tabs/motors.css:67]: If the purpose is an alignment with another thing, i cant see this aligment nowhere.

Comment (haslinghuis) [reply src/css/tabs/motors.css:67]: @limonspb likes to have the `mixerlist` as wide as the model image. `escprotocol` select is aligned with that. That means the mixerlist inhibits the same unnecessary extra whitespace.

Comment (Asizon) [reply src/css/tabs/motors.css:67]: I can see mixerlist aligned with modle image, its is good. But escprotocol is not aligned with anithing.

Comment (haslinghuis) [reply src/css/tabs/motors.css:67]: escprotocol is aligned with mixerlist

Comment (Asizon) [reply src/css/tabs/motors.css:67]: How align with mixer box if it is another diferent and separate item?

Comment (limonspb) [reply src/css/tabs/motors.css:67]: I think Asizon is correct. At least I wanted to alight only the mixer list with the image.

Comment (haslinghuis) [reply src/css/tabs/motors.css:67]: In that case we should also use `fit-content` on `mixerlist` to remove unnecessary extra whitespace.
Converting the list to uppercase will make the selector align with the `mixer preview img`

image

Comment (haslinghuis) [reply src/css/tabs/motors.css:67]: In that case we should also use `fit-content` on `mixerlist` to remove unnecessary extra whitespace.
Converting the list to uppercase will make the selector align with the `mixer preview img`

image

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/css/tabs/motors.css
Comment (limonspb) [reply src/css/tabs/motors.css:67]: i am not sure which is better actually. SO it is up to you @haslinghuis

**Review by limonspb — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
